wallet-new/src/Cardano/Wallet/API/Indices.hs

```diff
@@ -117,7 +121,7 @@ type family IndexToQueryParam resource ix where
     IndexToQueryParam Wallet  WalletId                = "id"
     IndexToQueryParam Wallet  (V1 Core.Timestamp)     = "created_at"
 
-    IndexToQueryParam WalletAddress (V1 Core.Address) = "id"
+    IndexToQueryParam WalletAddress (V1 Core.Address) = "address"
```

Comment:
NOTE: Wasn't used before, so there's no breaking change here. I believe `address` makes more sense even though it filters on the `id` field of an address. As a reminder, we represent addresses as follows:

![image](https://user-images.githubusercontent.com/5680256/42436833-8c9987fc-835b-11e8-97ca-c8e55475c257.png)

Reply:
man, that does feel really weird. i agree that this is an improvement, but this is confusing.